lib/utils/docs_parser/parse_docusaurus_docs.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:io';
+
+import 'package:path/path.dart';
+import 'package:solid_lints/utils/docs_parser/output_formatters/docusaurus_formatter.dart';
+import 'package:solid_lints/utils/docs_parser/parsers/docs_parser.dart';
+
+void main() async => DocsParser(
+      formatter: DocusaurusFormatter(),
+    ).parse(
+      Directory(
+        normalize(
+          join(Directory.current.path, 'lib', 'lints'),
+        ),
+      ),
+    );

```

Comment:
Let's add input-output parameters properly. For now we seem to be hardcoding a good lot, and it's really not obvious where things come from and where they go -- will make it tricky to figure out once we get back to this in some time.

Let's add some simple parameters that would make it apparent what the script takes in and what it outputs.